src/sentry/migrations/0261_prepare_remove_sentry_app_column.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 2.2.24 on 2021-12-10 21:12
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    # You'll also usually want to set this to `False` if you're writing a data
+    # migration, since we don't want the entire migration to run in one long-running
+    # transaction.
+    atomic = True
+
+    dependencies = [
+        ("sentry", "0260_backfill_integrationfeature"),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name="integrationfeature",
+            unique_together={("target_id", "target_type", "feature")},
+        ),
```

Comment:
Just for general awareness raising: Usually when adding an index to an existing table we will want to add it concurrently so we don't block operations. Since this is a small and low volume table we can skip it in this case.